integration_tests/mockApis/curiousApi.ts

```diff
@@ -316,6 +317,125 @@ const stubLearnerEducation = (prisonNumber = 'G6115VJ', page = 0): SuperAgentReq
     },
   })
 
+const stubLearnerEducationWithCoursesQualificationsInLast12Months = (
```

Comment:
I appreciate this could make for a very long method name, but I think the key word/phrase "completed" is missing here.
The important thing about this stub is that it is returning details of learner education (courses) that have been completed in the last 12 months.

I wonder if we can get this into the method name somehow without making it a massive long method name! 🤣